core/dive.c

```diff
@@ -1852,10 +1852,9 @@ static pressure_t merge_pressures(pressure_t a, pressure_t sample_a, pressure_t
 		a = b;
 	if (!b.mbar)
 		b = a;
-	if (take_min && a.mbar < b.mbar)
-		return a;
-	else
-		return b;
+	if(take_min)
+		return a.mbar < b.mbar? a : b;
```

Comment:
The commonly used form throughout Subsurface is `<condition> ? <true case> : <false case>` with a space before the `?`.

Reply:
Thank you; too many coding styles :)

I'll see if I can create a pre-commit hook to check the style (it's what my new employer does), although it might generate an extremely large number of changes depending on how unified the coding style is at present.

Reply:
> Thank you; fixed. Too many coding styles :)
> 
> I'll see if I can create a pre-commit hook to check the style (it's what my new employer does), although it might generate an extremely large number of changes depending on how unified the coding style is at present.

The coding style is different for different parts of the code. In part for historical reasons, in part to make merging with outside code easier, in part because we weren't always consistently enforcing it.

I appreciate it every time someone pays attention with what we intended to do.

Reply:
So there's already a `.clang-format` file in the repository; it would be trivial to run it on commit on the files which were changed. 

As far as current mis-match is concerned, we could always have a single pass over the entire codebase and fix up the style everywhere.  Although I must say I'm personally not a 100% fan of automated restyling tools - there are always situations in which hand-formatted code and comments are MUCH more readable even if it breaks the general coding guidelines/style.

Not sure if it's (easily) possible to have a different style for different parts of the codebase if maintaining merge-compatibility remains a requirement.